javascript/chronometer.js

```diff
@@ -1,26 +1,68 @@
 class Chronometer {
```

Comment:
Existe un pequeño bug, probad a dadle muy rápido a "start" / "stop". Veréis que cuando no llega a un segundo, se hace el lío luego hasta alcanzar un segundo. Ejemplo: si lo paráis en 0:800 cuando lo volvéis a arrancar. Tarda 1,2 s en llegar a marcar uno. Es un bug con el tiempo, una solución podría haber sido hacer que todo el currentTime estuviese basado en ms, así todo el cronómetro se basa en toda esa unidad en vez de tener distintos intervalos. No os preocupéis, es una mejora sobre el ejercicio. Era un bonus oculto 😉